report-service/sql-script/elr_validated.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE [NBS_DataIngest].[dbo].[elr_validated] (
     id UNIQUEIDENTIFIER PRIMARY KEY default NEWID(),
-    raw_id UNIQUEIDENTIFIER not null,
+    raw_id UNIQUEIDENTIFIER FOREIGN KEY REFERENCES [NBS_DataIngest].[dbo].[elr_raw](id),
```

Comment:
this could also be updated to raw_message_id like  elr_fhir

Reply:
Updated